### PR TITLE
[DOC] Edit the update-tutorials & get-demos script help texts & related NB doc

### PR DESCRIPTION
**Pull request by Sharon-iguazio**

### CODE REVIEW

File: get-additional-demos.sh [hunk 1]
```diff
@@ -14,16 +14,22 @@ user=${V3IO_USERNAME}
...
-  --dry-run   -  Do not update the files, but rather show the pending changes.
-  --no-backup -  Do not backup the current demos directory."
+  -b|--branch -  Git branch name. Default: The latest release branch that
+                 matches the version of the running MLRun service.
+  -u|--user   -  Username, which determines the directory to which to copy the
+                 retrieved demo files (/v3io/users/<username>).
+                 Default: \$V3IO_USERNAME, if set to a non-empty string.
+  --mlrun-ver -  The MLRun version for which to get demos; determines the Git
+                 branch from which to get the demos, unless -b|--branch is set.
+                 Default: The version of the running MLRun service.
```
Comment (gilad-shaham) [inline get-additional-demos.sh:28]: Same as above

File: get-additional-demos.sh [hunk 2]
```diff
@@ -14,16 +14,22 @@ user=${V3IO_USERNAME}
...
+  --dry-run   -  Show files to update but don't execute the update.
+  --no-backup -  Don't back up the existing demos directory before the update.
+                 Default: Back up the existing demos directory to a
+                 /v3io/users/<username>/demos.old.<timestamp>/ directory."
```
Comment (gilad-shaham) [inline get-additional-demos.sh:32]: Backup goes to "/v3io/users/\<username\>/demos.old/\<timestamp\>/"

File: get-additional-demos.sh [hunk 3]
```diff
@@ -14,16 +14,22 @@ user=${V3IO_USERNAME}
...
 OPTIONS:
   -h|--help   -  Display this message and exit.
-  -b|--branch -  Branch name (default is branch equivalent to the current MLRun version).
-  -u|--user   -  Username, which determines the directory to which to copy the files - /v3io/users/<username>.
-                 Default = \$V3IO_USERNAME (if set to a non-empty string).
-  --mlrun-ver -  Determine the branch based on the MLRun version, unless -b/--branch option is specified.
-  --dry-run   -  Do not update the files, but rather show the pending changes.
-  --no-backup -  Do not backup the current demos directory."
+  -b|--branch -  Git branch name. Default: The latest release branch that
+                 matches the version of the running MLRun service.
```
Comment (gilad-shaham) [inline get-additional-demos.sh:22]: Default is the version that matches the mlrun python package installed in the current python environment.

**Review by gilad-shaham — CHANGES_REQUESTED** [review]

File: get-additional-demos.sh (hunk 2 above)
Comment (Sharon-iguazio) [reply get-additional-demos.sh]: FIXED (changed '.' after "demos.old" to '/') — commit 3289bdc.

File: get-additional-demos.sh (hunk 3 above)
Comment (Sharon-iguazio) [reply get-additional-demos.sh]: I edited the help text for the `-b|--branch` and `--mlrun-ver` flags and the get-demos doc in **welcome.ipynb**/**README.md** — see commit 6f3279b9.

File: get-additional-demos.sh (hunk 1 above)
Comment (Sharon-iguazio) [reply get-additional-demos.sh]: Fixed — see my response to your `-b|--branch` comment — https://github.com/v3io/tutorials/pull/308#discussion_r547123857.

**Review by gilad-shaham — APPROVED** [review]

**Pull request merged**